src/modules/endpoint/sagas/loadCityContent.js

```diff
@@ -73,7 +79,7 @@ export default function * loadCityContent (
 
     const fetchMap = resourceURLFinder.buildFetchMap(
       categoriesMap.toArray().concat(events),
-      (url, path, urlHash) => buildResourceFilePath(url, path, newCity, urlHash)
+      (url, path, urlHash) => buildResourceFilePath(url, newCity, urlHash)
```

Comment:
You don't need `path` here

Reply:
I removed path here ;)

Reply:
It's still there in the arrow function o.O